File: src/main/java/ru/grabovsky/recordkeeping/models/dto/MessageDto.java
package ru.grabovsky.recordkeeping.models.dto;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.Instant;

@Data
@NoArgsConstructor
public class MessageDto {
    private String message;
    private Instant time = Instant.now();

    public MessageDto(String message) {
        this.message = message;
    }
}

File: api/src/main/kotlin/ru/grabovsky/recordkeeping/api/dto/utils/MessageDto.kt
package ru.grabovsky.recordkeeping.api.dto.utils

import com.fasterxml.jackson.annotation.JsonProperty
import io.swagger.v3.oas.annotations.media.Schema
import java.util.*

@Schema(description = "swagger.message.info", name = "Message")
data class MessageDto(
    @Schema(description = "swagger.message.info", example = "Информационное сообщение")
    @JsonProperty("message")
    val message: String
) {

    @Schema(description = "swagger.message.date")
    @JsonProperty("date")
    val date: Date = Date()
}
